src/components/list-item/list-item.scss

```diff
@@ -19,7 +19,7 @@
   );
 }
 .container:hover {
-  @apply bg-foreground-2;
+  @apply bg-foreground-2 cursor-pointer;
```

Comment:
👍 LOOKS GOOD.

I think we could probably remove this though...

https://github.com/Esri/calcite-components/blob/1fc4e760b8924ef8903f1283b7fa45e6d40e7231/src/components/list-item/list-item.scss#L51-L53